hub/api/dataset.py

```diff
@@ -254,10 +295,20 @@ def like(
 
         destination_ds.info.update(source_ds.info.__getstate__())  # type: ignore
 
+        if path.startswith("hub://"):
+            hub_reporter.feature_report(
+                feature_name="like",
+                parameters={"Path": str(path), "Overwrite": overwrite},
+            )
+        else:
+            hub_reporter.feature_report(
+                feature_name="like",
+                parameters={"Overwrite": overwrite},
+            )
```

Comment:
this block was repeated like 6 times, if statement and all. please generalize this and write a function (or better yet -- a decorator)

Reply:
@mccrearyd can we merge this as is for now. I don't know how to implement your proposal quickly, and we need to push this fix asap because it's a huge problem. Transforms are calling bugout reporting every sample, so we're getting 100k + reports per day.

Reply:
Or pls point me in the right direction on how to implement this as a decorator. Maybe some pseudocode and a place to put it.